Source/WebKit/UIProcess/AuxiliaryProcessProxy.h

```diff
@@ -246,7 +246,7 @@ AuxiliaryProcessProxy::SendSyncResult<T> AuxiliaryProcessProxy::sendSync(T&& mes
     static_assert(T::isSync, "Sync message expected");
 
     if (!m_connection)
-        return { };
+        return { IPC::Error::InvalidConnection };
```

Comment:
Thought (no change needed here) : Could that be the cause of our crashes? 🤔
But it shows this patch is doing its job already, and should prevent future similar mistakes.